2025-10-23 07:31:06,478 INFO    MainThread:33 [wandb_setup.py:_flush():81] Current SDK version is 0.22.2
2025-10-23 07:31:06,478 INFO    MainThread:33 [wandb_setup.py:_flush():81] Configure stats pid to 33
2025-10-23 07:31:06,478 INFO    MainThread:33 [wandb_setup.py:_flush():81] Loading settings from /root/.config/wandb/settings
2025-10-23 07:31:06,478 INFO    MainThread:33 [wandb_setup.py:_flush():81] Loading settings from /code/SHeRLoc_pr/training/wandb/settings
2025-10-23 07:31:06,478 INFO    MainThread:33 [wandb_setup.py:_flush():81] Loading settings from environment variables
2025-10-23 07:31:06,478 INFO    MainThread:33 [wandb_init.py:setup_run_log_directory():705] Logging user logs to /code/SHeRLoc_pr/training/wandb/run-20251023_073106-jcudoj9b/logs/debug.log
2025-10-23 07:31:06,478 INFO    MainThread:33 [wandb_init.py:setup_run_log_directory():706] Logging internal logs to /code/SHeRLoc_pr/training/wandb/run-20251023_073106-jcudoj9b/logs/debug-internal.log
2025-10-23 07:31:06,479 INFO    MainThread:33 [wandb_init.py:init():832] calling init triggers
2025-10-23 07:31:06,479 INFO    MainThread:33 [wandb_init.py:init():837] wandb.init called with sweep_config: {}
config: {'params_path': '../config/config.txt', 'model_params_path': '../models/Hetero_SHeRLoc.txt', 'dataset': 'hercules', 'dataset_folder': '/code/SHeRLoc/src/dataset/HeRCULES', 'num_workers': 0, 'batch_size': 20, 'lr': 0.001, 'epochs': 20, 'weight_decay': 0.001, 'loss': 'BatchHardTripletMarginLoss', 'margin': 1.0, 'train_dataset': '/code/SHeRLoc/src/dataset/HeRCULES', 'val_dataset': '/code/SHeRLoc/src/dataset/HeRCULES', 'test_dataset': '/code/SHeRLoc/src/dataset/HeRCULES', 'model_params.model_params_path': '../models/Hetero_SHeRLoc.txt', 'model_params.task': 'Hetero', 'model_params.radar_model': 'SHeRLoc', '_wandb': {}}
2025-10-23 07:31:06,479 INFO    MainThread:33 [wandb_init.py:init():880] starting backend
2025-10-23 07:31:06,870 INFO    MainThread:33 [wandb_init.py:init():883] sending inform_init request
2025-10-23 07:31:07,047 INFO    MainThread:33 [wandb_init.py:init():891] backend started and connected
2025-10-23 07:31:07,054 INFO    MainThread:33 [wandb_init.py:init():961] updated telemetry
2025-10-23 07:31:07,056 INFO    MainThread:33 [wandb_init.py:init():985] communicating run to backend with 90.0 second timeout
2025-10-23 07:31:19,810 WARNING MainThread:33 [wandb_init.py:init():1591] [no run ID] interrupted
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/wandb/sdk/wandb_init.py", line 1582, in init
    run = wi.init(run_settings, run_config, run_printer)
  File "/usr/local/lib/python3.9/site-packages/wandb/sdk/wandb_init.py", line 1004, in init
    result = wait_with_progress(
  File "/usr/local/lib/python3.9/site-packages/wandb/sdk/mailbox/wait_with_progress.py", line 23, in wait_with_progress
    return wait_all_with_progress(
  File "/usr/local/lib/python3.9/site-packages/wandb/sdk/mailbox/wait_with_progress.py", line 77, in wait_all_with_progress
    return asyncer.run(progress_loop_with_timeout)
  File "/usr/local/lib/python3.9/site-packages/wandb/sdk/lib/asyncio_manager.py", line 136, in run
    return future.result()
  File "/usr/local/lib/python3.9/concurrent/futures/_base.py", line 441, in result
    self._condition.wait(timeout)
  File "/usr/local/lib/python3.9/threading.py", line 312, in wait
    waiter.acquire()
KeyboardInterrupt
2025-10-23 07:31:19,826 INFO    wandb-AsyncioManager-main:33 [service_client.py:_forward_responses():80] Reached EOF.
2025-10-23 07:31:19,827 INFO    wandb-AsyncioManager-main:33 [mailbox.py:close():137] Closing mailbox, abandoning 2 handles.
